ember_debug/models/profile-manager.js

```diff
@@ -45,6 +118,16 @@ export default class ProfileManager {
     return callback.call(context);
   }
 
+  renderHighLight(view) {
+    const symbols = Object.getOwnPropertySymbols(view);
+    const bounds = view[symbols.find((sym) => sym.description === 'BOUNDS')];
```

Comment:
likely we should be shure that bounds is not undefined

Reply:
updated.